kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Serialization.java

```diff
@@ -56,6 +59,10 @@ public static ObjectMapper yamlMapper() {
     return YAML_MAPPER;
   }
 
+  public static Yaml snakeYamlMapper() {
+    return SNAKE_YAML_MAPPER;
+  }
```

Comment:
https://github.com/fabric8io/kubernetes-client/pull/2646/files#r540041655
```suggestion
  public static Yaml snakeYaml() {
    return new Yaml();
  }
```